src/root/DepositManager.ts

```diff
@@ -0,0 +1,69 @@
+import Web3 from 'web3'
+import { Contract } from 'web3/types'
+import { address, BigNumber, SendOptions } from '../types/Common'
+import DepositManagerArtifact from '../artifacts/DepositManager.json'
+
+export default class DepositManager {
+  private _depositManagerContract: Contract
+  private _defaultOptions: SendOptions
+
+  constructor(_depositManagerContractAddress: address, _web3: Web3, _defaultOptions?: SendOptions) {
+    this._depositManagerContract = new _web3.eth.Contract(DepositManagerArtifact.abi, _depositManagerContractAddress)
```

Comment:
Suggestion: better if we can use the variable as`_parentWeb3`.
more understandable for developers that while depositing you have use ParentWeb3 object
your call :D